ogre2/src/Ogre2MeshFactory.cc

```diff
@@ -382,13 +382,22 @@ bool Ogre2MeshFactory::LoadImpl(const MeshDescriptor &_desc)
 
       // two dimensional texture coordinates
       // add all texture coordinate sets
-      for (unsigned int k = 0u; k < subMesh.TexCoordSetCount(); ++k)
+      if (subMesh.TexCoordSetCount() == 0u)
```

Comment:
Consider adding a comment explaining why this is needed?

Reply:
added comment in [4985dbe](https://github.com/gazebosim/gz-rendering/pull/976/commits/4985dbeac5634b1014a623180a6ee2d8178ccfe4)